UPGRADE.md

```diff
@@ -168,6 +168,7 @@ After:
 
 * `ProductVariant::$name` property (and corresponding getter and setter) was removed to make it translatable. Therefore, `ProductVariantTranslation` was introduced with one `$name` property. All product variants names are migrated to new concept with migration `Version2016121415313`. Look at [this PR](https://github.com/Sylius/Sylius/pull/7091) if you have any problems with upgrade.
 * `ProductAssociationType::$name` property (and corresponding getter and setter) was removed to make it translatable. Therefore, `ProductAssociationTypeTranslation` was introduced with one `$name` property. All product association types names are migrated to new concept with migration `Version20161219160441`. Look at [this PR](https://github.com/Sylius/Sylius/pull/7134) if you have any problems with upgrade.
+* `Product:$avaiableOn` and `Product:$avaiableUntil`  properties (and corresponding getters and setters) were removed. All products are migrated to new concept with migration `Version20170208102345`. Look at [this PR](https://github.com/Sylius/Sylius/pull/7451) if you have any problems with upgrade.
```

Comment:
Anyway, I don't see any new concept in that migration, it just removes that fields. `to new concept` -> `to a new concept`